.github/workflows/nightly.yml

```diff
@@ -91,6 +90,33 @@ jobs:
           kubectl run --rm --attach test-http --image=galoisinc/cryptol-remote-api:test --image-pull-policy=Never --restart=Never -- http cra-http-cryptol-remote-api 8080
           kubectl run --rm --attach test-socket --image=galoisinc/cryptol-remote-api:test --image-pull-policy=Never --restart=Never -- socket cra-socket-cryptol-remote-api 8080
 
+  docker-cryptol-remote-api-portable:
```

Comment:
Building GHC with 2 cores takes an Extremely Long Time(tm), but its build can be cached against `ghcr.io/galoisinc/cryptol-remote-api:nightly-portable-cache`.

Reply:
I'm not familiar with buildx specifically, but I'm definitely in favor of caching stuff between CI runs and artifact creation whenever we can reliably do so.